### ISSUE TITLE
hello,can colmap support the line features triangulator?

### ISSUE THREAD

**Issue by zhengshunkai** [colmap/colmap#1570]
such as the title

**Comment by tsattler** [comment]
Colmap does not support line features or line feature triangulation.

**Comment by zhengshunkai** [comment]
> Colmap does not support line features or line feature triangulation.

Thx for your reply.by the way,have any plans to do this in the future？

**Comment by tsattler** [comment]
You will have to ask @ahojnnes , who is the developer of Colmap.

**Comment by ahojnnes** [comment]
No, there is no plan to support line features and it would likely require a fundamental refactor of the codebase. If somebody wants to add such a feature, I am open to discuss a plan to make that happen.

**Comment by yuancaimaiyi** [comment]
@zhengshunkai  sfm support line feature please refer to  【colmap 点线稀疏地图(在原来版本上改进增加线特征提取匹配，提高地图维度)】 https://www.bilibili.com/video/BV1dG411t7DN?share_source=copy_web&vd_source=2f60eca025dadfda6a988f2fafd67e50
line extracting  and matching refer to :https://github.com/cvg/SOLD2/

**Comment by SimonsRoad** [comment]
You can refer to this work in CVPR2023
https://github.com/cvg/limap 
Nice job！Thanks so much to these authors！
